code/modules/clothing/masks/gasmask.dm

```diff
@@ -154,6 +154,142 @@
 	item_state = "monkeymask"
 	flags_cover = MASKCOVERSEYES
 	resistance_flags = FLAMMABLE
+	actions_types = list(/datum/action/item_action/screech)
+	var/recent_uses = 0
+	var/broken_hailer = 0
+	var/gorilla = FALSE
+	var/cooldown_special = 0
+
+
+/obj/item/clothing/mask/gas/monkeymask/ui_action_click(mob/user, action)
+	screech()
+
+/obj/item/clothing/mask/gas/monkeymask/attack_self()
+	screech()
+/obj/item/clothing/mask/gas/monkeymask/emag_act(mob/user as mob)
+	if(!gorilla)
+		to_chat(user, "<span class='warning'>You silently turn on [src]'s gorilla module with the cryptographic sequencer.</span>")
+	else
+		return

```

Comment:
This actually makes me sick. Not only do you not set gorilla to true but you also have the completely harebrained
```
else
	return
```
statement after it.